src/stuf/stuf_bg/tests/test_client.py

```diff
@@ -244,6 +244,64 @@ def test_inp_heeftAlsKinderen(self):
         value = glom(data_dict, GlomTarget["inp.heeftAlsKinderen"], default=missing)
         self.assertNotEqual(value, missing)
 
+    @tag("gh-4338")
+    @requests_mock.Mocker()
+    def test_scattered_namespaces(self, m):
+        XML = b"""<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I guess we don't save it in another file and load it because it's the only one?I mean I am used to something like a `tests/files/xml_file_to_test` directory.

Reply:
yeah could have been an option but I strongly dislike needing to context switch when reading tests, and it's a super-specific regression test :shrug: